lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/models/domain/Status.scala

```diff
@@ -0,0 +1,18 @@
+package io.github.tetherlessworld.twxplore.lib.geo.models.domain
+
+sealed trait Status {
```

Comment:
The only problem with the sealed traits is that they're kind of a pain to represent in GraphQL. We may drop these fields in the API temporarily.